src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -696,7 +698,9 @@ private GameViewModel GetGameViewModelFor(int tournamentId)
             };
         }
 
-        private static List<SelectListItem> BuildTeamSelectList(Dictionary<int, List<TeamTournamentDto>> tournamentTeams, Dictionary<int, SelectListGroup> groups)
+#pragma warning disable S4017 // Method signatures should not contain nested generic types
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)